.circleci/config.yml

```diff
@@ -1164,6 +1167,87 @@ jobs:
             - notify-failures-on-develop:
                 mentions: "<<parameters.mentions>>"
 
+  op-acceptance-tests-docker:
```

Comment:
Necessary, because `op-acceptance-tests` runs in a VM with dynamic IPs, that don't have access to the endpoints we use in the op-node / op-sync-tester external tests.